### DISCUSSION TITLE
Switch between active groups.

### DISCUSSION THREAD

**Discussion by Magnus512** [qtile/qtile#3636] [Q&A] [answered]
Hello everyone, I have a question, for you.
Is there any way to swap between active groups ?
What I mean is, I have to monitors, and in I have one group in the left monitor and other in the right, but the active groups are not next to the other, by example group 1 in monitor 1 and group 4 in monitor 2, and I want to swap between this groups. 
I have proved screen.toggle_group() but that just swap between previous visited workspaces and I want to swap with any groups active on the two monitors.
I would appreciate any Idea,
![Windows](https://user-images.githubusercontent.com/7065012/174416894-3038e41d-fe33-487f-a143-f7847d989dbe.png)

**Comment by optimistic5** [comment, marked as answer]
Try to switch between monitors
```python
keys = [

    ### Switch focus to specific monitor (out of two)
    Key([mod], "e",
        lazy.to_screen(0),
        desc='Keyboard focus to monitor 1'
        ),
    Key([mod], "r",
        lazy.to_screen(1),
        desc='Keyboard focus to monitor 2'
        ),
    ### Switch focus of monitors
    Key([mod], "period",
        lazy.next_screen(),
        desc='Move focus to next monitor'
        ),
    Key([mod], "comma",
        lazy.prev_screen(),
        desc='Move focus to prev monitor'
        ),
        ...
]
```
